### PR TITLE
Add vision model sample to flutter_app

### PR DESCRIPTION
**Pull request by cynthiajoan**
Add a button to send prompt with text + two resource pictures to vision model

Screenshot 2024-02-15 at 11 31 05 AM

### CODE REVIEW

**Review by natebosch — COMMENTED** [review]
Does it work if we make the images symlinks?

**Review by natebosch — APPROVED** [review]
Looks good to me, but I don't have a background to evaluate the flutter usage.

**Comment by cynthiajoan** [comment]
> Does it work if we make the images symlinks?

Tried with symlinks and it doesn't work

File: samples/flutter_app/lib/main.dart [hunk 1]
```diff
@@ -67,12 +68,22 @@ class ChatWidget extends StatefulWidget {
   State<ChatWidget> createState() => _ChatWidgetState();
 }
 
+class ImageAndText {
```
[the remark this answers is not in the document]
Comment (cynthiajoan) [reply samples/flutter_app/lib/main.dart:71]: done

File: samples/flutter_app/lib/main.dart [hunk 2]
```diff
@@ -265,25 +341,27 @@ class MessageWidget extends StatelessWidget {
...
+                  horizontal: 20,
+                ),
+                margin: const EdgeInsets.only(bottom: 8),
+                child: Column(children: [
+                  if (text != null) ...[
+                    MarkdownBody(data: text!),
+                  ],
+                  if (image != null) ...[
+                    image!,
+                  ],
```
[the remark this answers is not in the document]
Comment (cynthiajoan) [reply samples/flutter_app/lib/main.dart:363]: done

**Pull request merged**